utils/lib/form.ex

```diff
@@ -66,19 +66,19 @@ defmodule Utils.Form do
     ]
 
     [
-      %{label: "concatenating two lists", answers: ["O(n1)"], options: options},
-      %{label: "inserting an element in tuple", answers: ["O(n)"], options: options},
-      %{label: "deleting an element in a list", answers: ["O(n*)"], options: options},
-      %{label: "prepending an element in a list", answers: ["O(1)"], options: options},
-      %{label: "updating an element in a list", answers: ["O(n*)"], options: options},
-      %{label: "concatenating two tuples", answers: ["O(n1 + n1)"], options: options},
-      %{label: "inserting an element in list", answers: ["O(n*)"], options: options},
-      %{label: "updating an element in tuple", answers: ["O(n)"], options: options},
-      %{label: "deleting an element in a tuple", answers: ["O(n)"], options: options},
-      %{label: "finding the length of a tuple", answers: ["O(1)"], options: options},
-      %{label: "deleting an element in a list", answers: ["O(n*)"], options: options},
-      %{label: "finding the length of a list", answers: ["O(n)"], options: options},
-      %{label: "finding the length of a list", answers: ["O(n)"]}
+      %{label: "Concatenating two lists", answers: ["O(n1)"], options: options},
```

Comment:
Great contribution! If you have time, we're attempting to replace `Utils` calls.
For example, this form could be in the new `HiddenCell` smart cell.

This is non-blocking, but thought I would mention it! LMK if you have any questions.